src/create-builder.js

```diff
@@ -119,12 +107,16 @@ module.exports = function bundler(userConfig, onDone) {
         if (config.watch) {
             bundles.forEach(bundlerTools.watchBundle);
         } else {
-            if (onDone) {
-                execOnDoneAfterFinished(map('bundle', bundles));
-            }
-            bundles.forEach(bundlerTools.writeBundle);
```

Comment:
this `forEach` meant that `writeBundle` would be called (_almost_) concurrently for all bundles, which could be many